// Copyright 2021 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// This file should be kept in sync with the same file in Google internal.
syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package cert_verify_tool;

message CertChain {
  optional string host = 1;
  repeated bytes der_certs = 2;
}